etc/ci/performance/test_all.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+case "$1" in
+  --servo)
+    engine=""
+    ;;
+  --gecko)
+    engine="--engine gecko"
+    ;;
+  *)
+    echo "Can't understand the argument, using the default servo engine"
+    echo "Usage: test_all.sh [--gecko]"
+    engine=""
+    ;;
+esac
+
+source venv/bin/activate
+
+echo "Starting the local server"
+python3 -m http.server > /dev/null 2>&1 &
+
+#MANIFEST="page_load_test/test.manifest"
+MANIFEST="page_load_test/20160509.manifest" # A manifest that excludes timeout test cases
+PERF_FILE="output/perf-$(date +"%s").json"
+
+echo "Running tests"
+python3 runner.py --runs 3 $MANIFEST $PERF_FILE 
```

Comment:
Please quote all variables and use the full "${variable}" form throughout the script, per the [style guide](https://github.com/servo/servo/blob/master/docs/STYLE_GUIDE.md).